src/version.ts

```diff
@@ -0,0 +1,2 @@
+export const version = '0.4.1';
```

Comment:
I would add a comment to this file saying that it was generated from a script.

Alternatively, say this in the file path. For instance: _src/generated/version.ts_